src/Avalonia.Themes.Fluent/Controls/ManagedFileChooser.xaml

```diff
@@ -179,8 +179,8 @@
               </DockPanel>
               <CheckBox IsChecked="{Binding ShowHiddenFiles}" Content="Show hidden files" DockPanel.Dock="Left"/>
               <UniformGrid x:Name="Finalize" HorizontalAlignment="Right" Rows="1">
-                <Button Command="{Binding Ok}">OK</Button>
-                <Button Command="{Binding Cancel}">Cancel</Button>
+                <Button Command="{Binding Ok}" Width="80">OK</Button>
```

Comment:
Don't use fixed Width where text may change in the future. As soon as Avalonia supports localization this is going to have to be translated.

It is better to set a MinWidth in cases like this.

Reply:
Switched to MinWidth